packages/x-date-pickers/src/internals/components/PickersPopper.tsx

```diff
@@ -49,13 +51,33 @@ export interface PickerPopperProps extends ExportedPickerPopperProps, ExportedPi
   onSetToday: () => void;
   components?: Partial<PickersPopperSlotsComponent>;
   componentsProps?: Partial<PickersPopperSlotsComponentsProps>;
+  classes?: Partial<PickersPopperClasses>;
 }
 
-const PickersPopperRoot = styled(Popper)<{ ownerState: PickerPopperProps }>(({ theme }) => ({
+const useUtilityClasses = (ownerState: PickerPopperProps) => {
+  const { classes } = ownerState;
+
+  const slots = {
+    root: ['root'],
+    paper: ['paper'],
+  };
+
+  return composeClasses(slots, getPickersPopperUtilityClass, classes);
+};
+
+const PickersPopperRoot = styled(Popper, {
+  name: 'MuiPickersPopper',
+  slot: 'Root',
+  overridesResolver: (_, styles) => styles.root,
+})<{ ownerState: PickerPopperProps }>(({ theme }) => ({
```

Comment:
If the owner state is not used, does it bring something to specify it in typescript?
```suggestion
})(({ theme }) => ({
```

Reply:
The `ownerState` prop was already being passed to this element, hence I did probably add this type here because of that, but you are correct, we can ditch this type and prop as they are not used anyways. 👍